tests/modules/msk/msisensor-ct/nextflow.config

```diff
@@ -0,0 +1,11 @@
+params {
+    enable_conda = false
+    fasta = "/home/buehlere/access_nextflow/getbasecountmultisample/chr22.fasta"
+    outdir = "output"
+}
+
+process {
```

Comment:
you can add optional parameters here with: `withName <module> { ext.args = '< option1 > < option2 >'} `